apps/nouns-camp/src/utils/transactions.js

```diff
@@ -394,7 +394,7 @@ export const unparse = (transactions) => {
         case "treasury-noun-transfer":
           return append({
             target: nounsTokenContract.address,
-            value: "",
+            value: "0",
```

Comment:
found this during testing. when a noun-transfer tx is submitted, its value becomes `0` onchain, so when editing a prop, there was a constant diff between what's onchain, and what we were unparsing.

Reply:
Pushed a fix for this on main fyi. Also found some more similar bugs. Great find!